.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": []
```

Comment:
It is some vscode config, but what does it do? Is it necessary?

Reply:
created automatically, if you want to add certain formatters or debuggers to the application